src/balatro_gym/cards/joker.py

```diff
@@ -120,3 +113,41 @@ def get_mult(self, scored_cards: Sequence[PlayingCard], state: BlindState, score
         if scored_hand == PokerHandType.THREE_SET:
             return 12
         return 0
+
+
+class MadJoker(JokerBase):
+    _base_cost: int = 4
+
+    @property
+    def joker_type(self) -> Type:
+        return Type.ADDITIVE_MULT
+
+    @property
+    def rarity(self) -> Rarity:
+        return Rarity.COMMON
+
+    def get_mult(self, scored_cards: Sequence[PlayingCard], state: BlindState, scored_hand: PokerHandType) -> int:
+        if scored_hand == PokerHandType.TWO_PAIR:
```

Comment:
For Mad, Jolly and Zany jokers, the joker is scored when the hand contains the hand, so here a full house should score the hand too, not just a two pair. I think we need to do the same adjustments for the other jokers

Reply:
Great catch, the latest commits should address this.